f/frizzle/alerts/alerts_gcs.py

```diff
@@ -379,106 +379,67 @@ def _table_exists(self, cursor, table_name):
+                    year text NOT NULL,
+                    total_alerts text NOT NULL,
+                    description_alerts text,
+                    confidence text,
+                    metadata_uuid text,
+                    source text,
+                    alert_source text
+                );
                 """
                 cursor.execute(query)
                 conn.commit()
-                logger.info(f"Table {table_name}__metadata created.")
-            else:
-                return
-            for column in metadata.keys():
-                # if the column is not one we care about do not add it
-                if column not in metadata_cols:
-                    continue
-                try:
-                    cursor.execute(f"""
-                    ALTER TABLE {table_name}__metadata
-                    ADD COLUMN "{column}" TEXT;
-                    """)
-                except errors.DuplicateColumn:
-                    logger.error(
-                        f"Skipping adding column due to DuplicateColumn, metadata column has been accounted for already in the past: {column}"
-                    )
-                    continue
-                except Exception as e:
-                    logger.error(
-                        f"An error occurred while creating new column: {column} for {table_name}__metadata: {e}"
-                    )
-                    raise
-
-    def _create_alert_cols(self, table_name):
-        # TODO: have this reflect the create_table.sql file
-        alert_cols = [
-            "_id",
-            "alert_source",
-            "alert_type",
-            "area_alert_ha",
-            "basin_id",
-            "count",
-            "date_end_t0",
-            "date_end_t1",
-            "date_start_t0",
-            "date_start_t1",
-            "grid",
-            "label",
-            "length_alert_km",
-            "month_detec",
-            "sat_detect_prefix",
-            "sat_viz_prefix",
-            "satellite",
-            "territory_id",
-            "territory_name",
-            "year_detec",
-            "g__coordinates",
-            "g__type",
-            "geom",
-            "source",
-        ]
+                logger.info(f"Table {metadata_table_name} created.")
 
+    def _create_alerts_table(self, table_name):
         with self._get_conn() as conn, conn.cursor() as cursor:
-            if not self._table_exists(cursor, f"{table_name}__metadata"):
+            if not self._table_exists(cursor, table_name):
                 query = f"""
-                CREATE TABLE public.{table_name}();
+                CREATE TABLE {table_name}
+                (
+                    _id character varying(36) NOT NULL,
+                    -- These are found in "properties" of an alert Feature:
+                    alert_type text,
+                    area_alert_ha double precision,
+                    basin_id bigint,
+                    count bigint,
```

Comment:
Can we add `confidence int`? I was just reviewing the GeoJSON files and seeing this is one that was added by the provider.

Reply:
Will do it in a separate PR

Reply:
https://github.com/ConservationMetrics/gc-scripts-hub/pull/38